### PR TITLE
Roberts Collection 翻译提交

### PR DESCRIPTION
**Pull request by HySand**

### CODE REVIEW

**Comment by luoxueyasha** [comment]
/sort-keys projects/1.19/assets/roberts-collection/roberts_collection/lang/en_us.json
/sort-keys projects/1.19/assets/roberts-collection/roberts_collection/lang/zh_cn.json

**Comment by luoxueyasha** [comment]
/sort-keys /projects/1.19/assets/roberts-collection/roberts_collection/lang/en_us.json

**Comment by luoxueyasha** [comment]
/sort-keys projects/1.19/assets/roberts-collection/roberts_collection/lang/en_us.json

**Comment by luoxueyasha** [comment]
/sort-keys projects/1.19/assets/roberts-collection/roberts_collection/lang/zh_cn.json

File: projects/1.19/assets/roberts-collection/roberts_collection/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  "block.roberts_collection.isaac_clarke": "艾萨克·克拉克",
+  "block.roberts_collection.joel_miller": "乔尔·米勒",
+  "block.roberts_collection.john_wick": "约翰·威克",
+  "block.roberts_collection.kratos": "奎托斯",
+  "block.roberts_collection.lara_croft": "劳拉·克罗夫特",
+  "block.roberts_collection.leon_s_kennedy": "里昂·斯科特·肯尼迪",
+  "block.roberts_collection.link": "林克",
+  "block.roberts_collection.mabel_pines": "梅宝·派恩斯",
+  "block.roberts_collection.makima": "玛奇玛",
+  "block.roberts_collection.mario_bros": "马力欧",
```
Comment (luoxueyasha) [inline projects/1.19/assets/roberts-collection/roberts_collection/lang/zh_cn.json:22]: 原文为`mario bros`
游戏里有路易吗

File: projects/1.19/assets/roberts-collection/roberts_collection/lang/zh_cn.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "block.roberts_collection.master_chief": "士官长",
+  "block.roberts_collection.mega_man": "洛克人",
+  "block.roberts_collection.pacman": "吃豆人",
+  "block.roberts_collection.pong": "乓",
+  "block.roberts_collection.power": "帕瓦",
+  "block.roberts_collection.princeof_persia": "波斯王子",
+  "block.roberts_collection.ryu": "隆",
+  "block.roberts_collection.samus_aran": "萨姆斯·阿兰",
+  "block.roberts_collection.simon_ghost_riley": "西蒙 \"幽灵\" 莱利",
```
Comment (luoxueyasha) [inline projects/1.19/assets/roberts-collection/roberts_collection/lang/zh_cn.json:31]:
```suggestion
  "block.roberts_collection.simon_ghost_riley": "西蒙·“幽灵”·莱利",
```
应使用汉语标点符号
查了一下，应当加入分隔符

File: projects/1.19/assets/roberts-collection/roberts_collection/lang/zh_cn.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  "block.roberts_collection.solairede_astora": "索拉尔",
```
Comment (luoxueyasha) [inline projects/1.19/assets/roberts-collection/roberts_collection/lang/zh_cn.json:32]:
```suggestion
  "block.roberts_collection.solairede_astora": "亚斯特拉的索拉尔",
```
原文为全称，故需要全文译出
<s>赞美太阳！</s>

**Review by luoxueyasha — CHANGES_REQUESTED** [review]
什么互联网亚文化大杂烩

File: projects/1.19/assets/roberts-collection/roberts_collection/lang/zh_cn.json (hunk 1 above)
Comment (HySand) [reply projects/1.19/assets/roberts-collection/roberts_collection/lang/zh_cn.json:22]: 只有马尿

**Review by luoxueyasha — APPROVED** [review]

**Pull request merged**